### PR TITLE
captchafy, not capatchafy

### PR DESCRIPTION
**Pull request by ajaxvp**
i fixed the most rage-worthy mistake in the history of mankind.

### CODE REVIEW

**Comment by script-head** [comment]
how did you manage to make it rewrite every file

**Comment by ajaxvp** [comment]
> how did you manage to make it rewrite every file

i did it manually, every single one of them

File: .idea/misc.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="9" project-jdk-type="JavaSDK">
```
Comment (lemonsked) [inline .idea/misc.xml]: change back, we don't need to use java 9 features

File: .idea/uiDesigner.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      </item>
+      <item class="javax.swing.JToolBar" icon="/com/intellij/uiDesigner/icons/toolbar.png" removable="false" auto-create-binding="false" can-attach-label="false">
+        <default-constraints vsize-policy="0" hsize-policy="6" anchor="0" fill="1">
+          <preferred-size width="-1" height="20" />
+        </default-constraints>
+      </item>
+      <item class="javax.swing.JToolBar$Separator" icon="/com/intellij/uiDesigner/icons/toolbarSeparator.png" removable="false" auto-create-binding="false" can-attach-label="false">
+        <default-constraints vsize-policy="0" hsize-policy="0" anchor="0" fill="1" />
+      </item>
+      <item class="javax.swing.JScrollBar" icon="/com/intellij/uiDesigner/icons/scrollbar.png" removable="false" auto-create-binding="true" can-attach-label="false">
```
Comment (lemonsked) [inline .idea/uiDesigner.xml]: what

**Review by lemonsked — DISMISSED** [review]

File: .idea/uiDesigner.xml (hunk 2 above)
Comment (ajaxvp) [reply .idea/uiDesigner.xml]: all done boss

File: .idea/misc.xml (hunk 1 above)
Comment (ajaxvp) [reply .idea/misc.xml]: all done boss

**Comment by ajaxvp** [comment]
can this be merged, @OxLemonxO?

**Comment by robingall2910** [comment]
ok, @SuperSteelYT

**Pull request merged**